### PR TITLE
Add integration tests for issues: 1685, 1686, 1687

### PR DESCRIPTION
**Pull request by MacondoExpress**
# Description

Add integrations test for connections fields and queries.

### CODE REVIEW

File: packages/graphql/tests/integration/issues/1687.int.test.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+ * limitations under the License.
+ */
+
+import { graphql, GraphQLSchema } from "graphql";
+import { Driver } from "neo4j-driver";
+import Neo4j from "../neo4j";
+import { Neo4jGraphQL } from "../../../src";
+import { generateUniqueType } from "../../utils/graphql-types";
+
+describe("https://github.com/neo4j/graphql/issues/1686", () => {
```
Comment (tbwiss) [inline packages/graphql/tests/integration/issues/1687.int.test.ts:26]:
```suggestion
describe("https://github.com/neo4j/graphql/issues/1687", () => {
```

**Review by tbwiss — APPROVED** [review]
LGTM!
Just one nit-picking suggestion.
And review-dog doesn't like the `test.skip` which is expected..

File: packages/graphql/tests/integration/issues/1687.int.test.ts [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+    test.skip("should be able to returns all the genres related to Matrix using connection fields", async () => {
+        const query = `
+        query Genres {
+            ${genreType.plural}(where: {
+              ${movieType.operations.connection}_SOME: {
+                node: {
+                  _on: {
+                    ${movieType.name}: {
+                      title: "The matrix"
```
Comment (tbwiss) [inline packages/graphql/tests/integration/issues/1687.int.test.ts:82]:
```suggestion
                      title: "Matrix"
```

**Comment by MacondoExpress** [comment]
> @MacondoExpress, have all of these tests now been included in bugfix branches? Can we close this?

Yes, these tests were added directly to the relative fix branches

**Pull request closed without merging**